### ISSUE TITLE
EMailSender works only with smtp.gmail.com port 465

### ISSUE THREAD

**Issue by lerman06470** [xreef/EMailSender#30]
By default, EMailSender uses [smtp.gmail.com](http://smtp.gmail.com/) and port 465. That's coded into EMailSender.h.
I tried to use [smtp.sendgrid.net](http://smtp.sendgrid.net/), port 25 by calling:
```  emailSend.setSMTPServer(smtpServer);
    emailSend.setSMTPPort(smtpPort);
```
The platform is a nano 33 IOT.

I ran tcpdump on my firewall and found that some invalid stuff is sent prior to the HELO. It appears to be a TLS handshake which gmail handles just fine. I tried turning off isSecure:
```emailSend.setIsSecure(false);```
That didn't seem to help, and tcpdump still showed extraneous data being sent.

Using a secure port on sendgrid also failed to connect.

**Comment by xreef** [comment]
Hi,
search on the forum, there is a topic that describe all the step to configure send grid.
Bye Renzo

**Comment by lerman06470** [comment]
I looked, but didn't find anything useful. The underlying problem should be
addressed or documented.

Looking at the code I found that on this platform EMailSender ignores the
isSecure flag.

I patched the code to use WiFiClient instead of the SSL version since that
solves my immediate problem.

I'll open a new issue so that it doesn't get lost.

Thanks,

Ken

Kenneth Lerman
55 Main Street
Newtown, CT 06470
